.travis.yml

```diff
@@ -14,19 +14,34 @@ install:
   - docker build --build-arg BUILDKIT_INLINE_CACHE=1 --cache-from j0elwee/devops-trg:python-3.8-slim-buster-with-geckodriver -t j0elwee/devops-trg:python-3.8-slim-buster-with-geckodriver -f Dockerfile.test.base .
 script:
   - docker-compose -f docker-compose.test.yml up --exit-code-from tests
+
 before_deploy:
   - echo $DOCKER_HUB_PASSWORD | docker login -u j0elwee --password-stdin
   - docker push j0elwee/devops-trg
   - docker push j0elwee/devops-trg:$TRAVIS_COMMIT
   - docker push j0elwee/devops-trg:python-3.8-slim-buster-with-geckodriver
+  - wget https://releases.hashicorp.com/terraform/"$TF_VERSION"/terraform_"$TF_VERSION"_linux_amd64.zip
+  - unzip terraform_"$TF_VERSION"_linux_amd64.zip
+  - sudo mv terraform /usr/local/bin/
+  - rm terraform_"$TF_VERSION"_linux_amd64.zip
+  - terraform init
+  - terraform apply -auto-approve -var "oauth_client_id=$OAUTH_CLIENT_ID" -var "oauth_client_secret=$OAUTH_CLIENT_SECRET" -var "secret_key=$SECRETKEY"
```

Comment:
Your SECRET_KEY setting is coming through blank in your app (which causes errors) - I wonder if $SECRETKEY here doesn't match the variable name you've set in travis? (Unfortunately it's hard to see that because Travis thinks one of your variables needs to have the key blocked out because it matches a secure value; I suspect you've set something like `SECRETKEY=SECRETKEY`)